tests/requirements/requirements_base.txt

```diff
@@ -1,8 +1,10 @@
+beautifulsoup4
 coverage
 django-app-helper
 flake8
 factory_boy
 isort
+lxml
```

Comment:
```suggestion
```

lxml won't be required with beautiful soup, it's one or the other unless you are explicitly using it.